hack/common.sh

```diff
@@ -115,3 +116,15 @@ get_version() {
 
   echo $VERSION
 }
+
+# Given two parameters, replaces the image value in the first parameter in the csv with the other
```

Comment:
The sed command will replace any value in the CSV and not just the image value. I am fine if you just fix the comment and function name. `transform_csv`.